packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-submission-for-mt5.jsx

```diff
@@ -16,22 +16,19 @@ const POISubmissionForMT5 = ({
     refreshNotifications,
     citizen_data,
     has_idv_error,
-    jurisdiction_selected_shortcode,
+    residence_list,
+    account_settings,
+    is_eu_user,
 }) => {
     const [submission_status, setSubmissionStatus] = React.useState(); // submitting
     const [submission_service, setSubmissionService] = React.useState();
     React.useEffect(() => {
         if (citizen_data) {
             const { submissions_left: idv_submissions_left } = idv;
             const { submissions_left: onfido_submissions_left } = onfido;
-            const is_idv_supported = citizen_data.identity.services.idv.is_country_supported;
-            const is_onfido_supported = citizen_data.identity.services.onfido.is_country_supported;
-            if (
-                is_idv_supported &&
-                Number(idv_submissions_left) > 0 &&
-                !is_idv_disallowed &&
-                jurisdiction_selected_shortcode !== 'vanuatu'
-            ) {
+            const is_idv_supported = isVerificationServiceSupported(residence_list, account_settings, 'idv');
+            const is_onfido_supported = isVerificationServiceSupported(residence_list, account_settings, 'onfido');
+            if (is_idv_supported && Number(idv_submissions_left) > 0 && !is_idv_disallowed && !is_eu_user) {
```

Comment:
cant we use jurisdiction_selected_shortcode !== 'maltainvest' instead of !is_eu_user?

Reply:
this flag is being used to identity in traders_hub if the user is in EU regulation or Non-EU and its well tested & works fine with both IDV & onfido supported countries

Reply:
eu users will have only maltainvest as available jurisdiction

Reply:
Yes. maltainvest will always trigger onfido